source/NVDAObjects/UIA/__init__.py

```diff
@@ -2055,6 +2055,35 @@ def event_UIA_itemStatus(self):
 			ui.message(f"{self.name} {itemStatus}")
 		self._itemStatusCache = itemStatus
 
+	def event_UIA_dragDropEffect(self):
+		# UIA drag drop effect was introduced in Windows 8.
+		try:
+			ui.message(self._getUIACacheablePropertyValue(UIAHandler.UIA_DragDropEffectPropertyId))
+		except COMError:
+			pass
+
+	def event_UIA_dropTargetEffect(self):
+		# UIA drop target effect property was introduced in Windows 8.
+		try:
+			dropTargetEffect = self._getUIACacheablePropertyValue(
+				UIAHandler.UIA_DropTargetDropTargetEffectPropertyId
+			)
+		except COMError:
+			dropTargetEffect = ""
+		# Sometimes drop target effect text is empty as it comes from a different object.
+		if not dropTargetEffect:
+			for element in reversed(api.getFocusAncestors()):
+				try:

```

Comment:
Please avoid calling _getUIACacheablePropertyValue on NON-UIA objects.

```suggestion
				if not isinstance(element, NVDAObjects.UIA.UIA):
					continue
				try:
```